### PR TITLE
Icons: Added/updated

### PR DESCRIPTION
**Pull request by swasunb**
This PR updates 3 icons and adds 4 icona.

Before:

Snip20210928_3

After:
Snip20210929_4

### CODE REVIEW

**Comment by symphony-adnane** [comment]
@swasunb did we remove "video" icon?

**Comment by swasunb** [comment]
> @swasunb did we remove "video" icon?

renamed it to use video-on

**Review by liuting1014 — APPROVED** [review]

**Comment by symphony-adnane** [comment]
@swasunb that's gonna be a breaking change, could we keep the old?

**Review by symphony-adnane — APPROVED** [review]

File: src/icons/svg/mic-on.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M7.75 0C5.54086 0 3.75 1.79086 3.75 4V7C3.75 9.20914 5.54086 11 7.75 11C9.95914 11 11.75 9.20914 11.75 7V4C11.75 1.79086 9.95914 0 7.75 0ZM9.75 7C9.75 8.10457 8.85457 9 7.75 9C6.64543 9 5.75 8.10457 5.75 7V4C5.75 2.89543 6.64543 2 7.75 2C8.85457 2 9.75 2.89543 9.75 4V7Z" fill="black"/>
-<path d="M1.75 6.25C2.16422 6.25 2.5 6.58579 2.5 7C2.5 9.89949 4.85051 12.25 7.75 12.25C10.6495 12.25 13 9.89949 13 7C13 6.58579 13.3358 6.25 13.75 6.25C14.1642 6.25 14.5 6.58579 14.5 7C14.5 10.3881 12.0037 13.1932 8.75 13.6764V15C8.75 15.5523 8.30229 16 7.75 16C7.19771 16 6.75 15.5523 6.75 15V13.6764C3.49629 13.1932 1 10.3881 1 7C1 6.58579 1.33578 6.25 1.75 6.25Z" fill="black"/>
+<svg width="15" height="17" viewBox="0 0 15 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (symphony-adnane) [inline src/icons/svg/mic-on.svg:1]: I think we need to keep the 16*16 ratio

Comment (swasunb) [reply src/icons/svg/mic-on.svg:1]: I think we have other icons, where we do not have 16x16

Comment (swasunb) [reply src/icons/svg/mic-on.svg:1]: But may be i can check with Yammie Seto to keep it 16*16

**Review by symphony-annaLlorens — APPROVED** [review]

File: src/icons/svg/mic-on.svg (hunk 1 above)
Comment (swasunb) [reply src/icons/svg/mic-on.svg]: @symphony-adnane updated.

**Pull request merged**